### PR TITLE
Migrate rest of the badges in admin UI

### PR DESCRIPTION
**Pull request by DhariniJeeva**
[jira](https://hashicorp.atlassian.net/browse/ICU-6600)

### CODE REVIEW

File: ui/admin/app/components/host-catalog-type-badge/index.hbs [hunk 1]
```diff
@@ -1,15 +1,4 @@
...
-{{else}}
-  <Rose::Badge
-    @icon={{concat 'flight-icons/svg/' @model.compositeType '-color-16'}}
-  >
-    {{t (concat 'resources.host-catalog.types.' @model.compositeType)}}
-  </Rose::Badge>
-{{/if}}
\ No newline at end of file
+<Hds::Badge
+  @icon={{this.icon}}
```
Comment (randallmorey) [inline ui/admin/app/components/host-catalog-type-badge/index.hbs:2]: Does this work if icon is falsy?

File: ui/admin/tests/acceptance/credential-store/read-test.js [hunk 2]
```diff
@@ -83,18 +83,6 @@ module('Acceptance | credential-stores | read', function (hooks) {
...
-    await visit(urls.orgScope);
-    instances.scopes.project.authorized_collection_actions[
-      'credential-stores'
-    ] = [];
-
-    await click(`[href="${urls.projectScope}"]`);
-
-    assert.dom(`[href="${urls.credentialStores}"]`).doesNotExist();
-  });
-
```
Comment (randallmorey) [inline ui/admin/tests/acceptance/credential-store/read-test.js:97]: Why is this removed?

**Review by randallmorey — APPROVED** [review]
Looks good.  Left some questions.

File: ui/admin/tests/acceptance/credential-store/read-test.js (hunk 2 above)
Comment (DhariniJeeva) [reply ui/admin/tests/acceptance/credential-store/read-test.js:97]: Don't think its part of this PR. Let me check and rebase with the main!

File: ui/admin/app/components/host-catalog-type-badge/index.hbs (hunk 1 above)
Comment (randallmorey) [reply ui/admin/app/components/host-catalog-type-badge/index.hbs:2]: Why are Azure and AWS without icons here?  Was this just an example?

Comment (DhariniJeeva) [reply ui/admin/app/components/host-catalog-type-badge/index.hbs:2]: yes, an example of when I passed in a falsy value.

**Review by ZedLi — APPROVED** [review]

**Review by calcaide — APPROVED** [review]
LGTM! Awesome work!!
